code/lib/cli/src/generators/configure.test.ts

```diff
@@ -193,21 +208,27 @@ describe('configurePreview', () => {
 
     expect(previewConfigPath).toEqual('./.storybook/preview.ts');
     expect(previewConfigContent).toMatchInlineSnapshot(`
-      "import { setCompodocJson } from \\"@storybook/addon-docs/angular\\";
+      "import type { Preview } from '@storybook/angular'
```

Comment:
I think the end result should be like the other import?

```
import type { Preview } from \\"@storybook/angular\\"
```

Reply:
Hm, maybe we should run prettier on it, using user config, if possible. The `main.js` template you wrote is actually using single quotes everywhere:

```
  if (isTypescript) {
    imports.push(`import type { StorybookConfig } from '${frameworkPackage}';`);
  } else {
    imports.push(`/** @type { import('${frameworkPackage}').StorybookConfig } */`);
  }
```

Reply:
Prettified the files based on user config